config/config.yaml

```diff
@@ -1,6 +1,12 @@
+# Server configurations
```

Comment:
Can we rename this file to be used for dev purpose. Like we have different files for each environment.